### ISSUE TITLE
The battery charge bar is flickering on my iphone 13 in drive mode.

### ISSUE THREAD

**Issue by Tho-Mat** [Kemmey/Kemmey-TeslaWatch-Public#497]
In drive mode the green status of the battery is fickering. 
My be you should not refesh the bar that often.
iPhone 13 pro max, ios16

**Comment by Kemmey** [comment]
Yeah - it's a swiftui bug in iOS 16 / watchOS 9. I'll try to find a workaround
